spec/ruby_terraform/commands/init_spec.rb

```diff
@@ -11,120 +14,23 @@
     RubyTerraform.reset!
   end
 
-  it 'calls the terraform init command' do
-    command = RubyTerraform::Commands::Init.new(binary: 'terraform')
-
-    expect(Open4).to(
-        receive(:spawn)
-            .with('terraform init', any_args))
-
-    command.execute
-  end
-
-  it 'defaults to the configured binary when none provided' do
-    command = RubyTerraform::Commands::Init.new
-
-    expect(Open4).to(
-        receive(:spawn)
-            .with('path/to/binary init', any_args))
-
-    command.execute
-  end
-
-  it 'includes the no-color flag when the no_color option is true' do
-    command = RubyTerraform::Commands::Init.new
-
-    expect(Open4).to(
-        receive(:spawn)
-            .with('path/to/binary init -no-color', any_args))
-
-    command.execute(no_color: true)
-  end
-
-  it 'includes the force-copy flag when the force_copy option is true' do
-    command = RubyTerraform::Commands::Init.new
+  terraform_command = 'init'
 
-    expect(Open4).to(
-        receive(:spawn)
-            .with('path/to/binary init -force-copy=true', any_args))
+  it_behaves_like 'a command without a binary supplied', [terraform_command, described_class, { directory: nil }]
```

Comment:
Would it be better to require that `directory` is supplied always than to have to pass `nil` when it's not required?